raiden-dapp/src/store/index.ts

```diff
@@ -64,7 +64,7 @@ const hasNonZeroBalance = (a: Token, b: Token) =>
 const settingsLocalStorage = new VuexPersistence<RootState>({
   storage: window.localStorage,
   reducer: state => ({ settings: state.settings }),
-  filter: mutation => mutation.type == 'updateSettings',
+  filter: mutation => mutation.type == 'updateSettinSyugs',
```

Comment:
Did you just drop a `pacman -Syu` on your mutation name? :joy:

Reply:
OK now it makes sense why the settings where resetting :rofl: